components/dashboard/src/settings/Account.tsx

```diff
@@ -12,15 +12,18 @@ export default function Account() {
     const close = () => setModal(false);
     return <div>
         <Modal visible={modal} onClose={close}>
-            <h3>Do you really want to delete your account?</h3>
-            <p>This action will remove all the data associated with your account in Gitpod and cannot be reversed.</p>
-            <div className="flex justify-end pt-6">
+            <h3 className="pb-2">Delete Account</h3>
+            <div className="border-t border-b border-gray-200 mt-2 -mx-6 px-6 py-2">
+              <p className="mt-1 mb-2 text-base">Are you sure you want to delete your account? This action will remove all the data associated with your account in Gitpod and cannot be reversed.</p>
+            </div>
+            <div className="flex justify-end mt-6">
                 <button className="border-red-900 bg-red-500 hover:bg-red-700" onClick={close}>Delete Account</button>
             </div>
         </Modal>
 
-        <SettingsPage title='Account' subtitle='Profile details'>
+        <SettingsPage title='Account' subtitle='Manage account and git configuration.'>
             <h3>Profile</h3>
+            <p className="text-base text-gray-500 pb-4">The following information will be used to set up git configuration. You can override git author name and email per project by using the default environment variables <span className="text-gitpod-kumquat-dark bg-gitpod-kumquat-light px-1.5 py-1 rounded-md text-sm font-mono">GIT_AUTHOR_NAME</span> and <span className="text-gitpod-kumquat-dark bg-gitpod-kumquat-light px-1.5 py-1 rounded-md text-sm font-mono">GIT_COMMITTER_EMAIL</span>.</p>
```

Comment:
**issue**: Name still is not picking up name when the user has only authorized with a GL provider.

Reply:
Resolving, added follow up issue in https://github.com/gitpod-io/gitpod/issues/3532.